packages/dcos-integration-test/extra/test_dcos_log.py

```diff
@@ -0,0 +1,100 @@
+import json
+
+import requests
+
+
+def validate_json_entry(entry: dict):
+    required_fields = set(['fields', 'cursor', 'monotonic_timestamp', 'realtime_timestamp'])
+
+    assert set(entry.keys()) <= required_fields, (
+        "Entry didn't have all required fields. Entry fields: {}, required fields:{}".format(entry, required_fields))
+
+    assert entry['fields'], '`fields` cannot be empty dict. Got {}'.format(entry)
+
+
+def validate_sse_entry(entry, stream=False):
+    lines = entry.split('\n')
+    assert len(lines) >= 2, 'Expect an least 2 lines of entry. Got {}'.format(entry)
+
+    data_line = {
+        True: lines[1],
+        False: lines[0]
+    }.get(stream)
+
+    entry_json = json.loads(data_line.lstrip('data: '))
+    validate_json_entry(entry_json)
+
+    if stream:
+        cursor = lines[0].lstrip('id: ')
+        assert cursor == entry_json['cursor'], 'In streaming mode an entry must have id: Got {}'.format(entry)
+
+
+def check_response_ok(response: requests.models.Response, headers: dict):
+    assert response.ok, 'Request {} returned response code {}'.format(response.url, response.status_code)
+    for name, value in headers.items():
+        assert response.headers.get(name) == value, (
+            'Request {} header {} must be {}. All headers {}'.format(response.url, name, value, response.headers))
+
+
+def test_log_text(cluster):
+    for node in cluster.masters + cluster.all_slaves:
+        response = cluster.logs.get('v1/range/?limit=10', node=node)
+        check_response_ok(response, {'Content-Type': 'text/plain'})
+
+        # expect 10 lines
+        lines = filter(lambda x: x != '', response.content.decode().split('\n'))
+        assert len(list(lines)) == 10, 'Expect 10 log entries. Got {}. All lines {}'.format(len(lines), lines)
```

Comment:
nit: Would be nice if we could check the text is something sensical / not jumbled bytes, but this works well enough for now. Should be doable once this gets to collecting container logs (Launch a container that prints a couple of very specific lines into the log, make sure that in the overall list of log lines those lines appear in the right order.